browse/domain/version.py

```diff
@@ -115,6 +115,9 @@ class VersionEntry:
     source_type: SourceType = field(default_factory=SourceType)  # type: ignore
     """Source file type."""
 
+    is_withdrawn: bool = False
```

Comment:
DB has a separate is_withdrawn column. .abs files store this in the source flags.